olwebsite-app/src/components/Toolbar/__tests__/HamburgerToolbar.test.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { render, fireEvent, getByLabelText } from '@testing-library/react';
```

Comment:
Check out the test-utils.js file. The testing library is setup so that you don't have to directly import   from the testing-library package. The whole testing-library is exported there as well as an additional utility function that allows you to render a component with router. You can import it like below:

```javascript
import { render, fireEvent, getByLabelText, renderWithRouter } from "test-utils";
```